packages/starlight/__e2e__/fixtures/ssr/src/content/docs/content.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: Tabs
```

Comment:
Let's update this fixture page title to avoid mentioning unrelated content from another test.